lib/system_description.rb

```diff
@@ -111,19 +111,21 @@ def from_hash(name, store, hash)
 
     private
 
-    def create_scopes(hash)
-      scopes = hash.map do |scope_name, scope_json|
-        next if scope_name == "meta"
+    def create_scopes(store, name, hash)
+      scopes = hash.map do |scope, json|
+        next if scope == "meta"
 
-        scope_class = Machinery::Scope.class_for(scope_name)
-        scope_object = scope_class.from_json(scope_json)
+        if store.persistent?
+          scope_file_store = ScopeFileStore.new(store.description_path(name), scope)
```

Comment:
We do, but only as an instance method. This code is defined on the class instead.